src/integrate.ts

```diff
@@ -13,14 +13,7 @@ function isOutputFn(output: OutputFn | OutputFnDelayed): output is OutputFn {
     return output === null || output.length === 2;
 }
 
-function isRhsFnDelayed(rhs: RhsFn | RhsFnDelayed): rhs is RhsFnDelayed {
```

Comment:
If this is not used any more, does the error message below still make sense "Can't use non-delayed output with delayed rhs"? Or are all rhses delayed here?

Reply:
It's not used anywhere because we do it with `if isRhsFn / else` as that keeps the compiler happier